src/xunit.runner.uap/StandardUapVisitor.cs

```diff
@@ -0,0 +1,190 @@
+using System;
+using System.Collections.Concurrent;
+using System.IO;
+using System.Text;
+using System.Xml.Linq;
+using Xunit.Abstractions;
+
+namespace Xunit.ConsoleClient
+{
+    public class StandardUapVisitor : XmlTestExecutionVisitor
+    {
+        string assemblyName;
+        readonly ConcurrentDictionary<string, ExecutionSummary> completionMessages;
+        readonly StreamWriter log;
+        readonly bool showProgress;
+        readonly bool failSkips;
```

Comment:
could you follow the code notations we use? I mean put "private" specifier and would be good to name the fields prefixed with underscore so you don't have to use "this.xxx" while accessing it

Reply:
Yes, forgot to update that :)